### DISCUSSION TITLE
[Heroicons]: Mirroring the refresh icon to work with Tailwind's animate-spin

### DISCUSSION THREAD

**Discussion by jessarcher** [tailwindlabs/tailwindcss#7599] [Help]
Thanks for the amazing work!

Currently the direction of the arrows on the refresh icon points anti-clockwise, so when combined with Tailwind's `animate-spin` utility, it appears to be spinning backwards.

If this was mirrored (which is admittedly easy enough to do ad-hoc) it would fix the issue.

I'm not too sure how to PR this with the Figma file, optimized directory, etc.

Thanks!

**Comment by devinfd** [comment]
Yes! This is hurting my head

**Comment by simonswiss** [comment]
Hey!

Totally a workaround, but you can wrap your SVG in an element, and have the `animate-spin` class on that element.

Then, on the SVG itself, set a `scale-x-[-1]` class (which will basically flip the icon horizontally.

https://play.tailwindcss.com/AqpZaEVw0E

The reason you need the wrapping element is that if you apply `animate spin` on the SVG directly (alongside the "flip" class), the `transform` properties in both those utilities will clash.

Hope it helps!

I'm not saying there will never be an "inverted" refresh icon in Heroicons – but at least you've got a working solution in the meantime 👍

Will move this issue to our GitHub discussions, as some other folks may find this trick useful!
